src/BlockchainWriter/BlockInfoDAO.ts

```diff
@@ -0,0 +1,26 @@
+import { LightBlock } from 'Messaging/Messages'
+import { inject, injectable } from 'inversify'
+import { Collection } from 'mongodb'
+
+@injectable()
+export class BlockInfoDAO {
+  private readonly blockchainInfo: Collection
+
+  constructor(
+    @inject('BlockchainInfoCollection') blockchainInfo: Collection,
+  ) {
+    this.blockchainInfo = blockchainInfo
+  }
+
+  readonly start = async () => {
+    await this.blockchainInfo.createIndex({ height: 1, hash: 1 }, { unique: true })
+  }
+
+  readonly insertBlockInfo = (lightBlock: LightBlock) =>
+    this.blockchainInfo.insertOne(lightBlock)
+
+  readonly getMaxHeightBlock = async (): Promise<LightBlock> => {
+    const maxBlock = this.blockchainInfo.find().sort({height: -1}).limit(1)
```

Comment:
For slightly better perf:

```suggestion
    const maxBlock = this.blockchainInfo.find({}, { height: 1, hash: 1, _id: 0 }).sort({height: -1}).limit(1)
```

For details see the comment of https://stackoverflow.com/a/32077240 and their link to:
https://docs.mongodb.com/v3.0/core/query-optimization/#covered-query